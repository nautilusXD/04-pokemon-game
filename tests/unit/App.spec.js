import { shallowMount } from "@vue/test-utils";
import App from "@/App";

describe("App", () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallowMount(App);
    });

    test("Debe de coincidir con el snapshot", () => {
        expect(wrapper.html()).toMatchSnapshot();
    });
});