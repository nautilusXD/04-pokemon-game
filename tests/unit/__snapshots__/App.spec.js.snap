// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Debe de coincidir con el snapshot 1`] = `<pokemon-page-stub></pokemon-page-stub>`;
